Handle case when `configure.sdkroot` has no version number
    (e.g. `configure.sdkroot` is /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk).

--- setup.py.orig	2019-10-16 02:40:21.000000000 -0700
+++ setup.py	2021-04-14 11:13:42.000000000 -0700
@@ -58,7 +58,10 @@
     assert sdk.startswith('MacOSX')
     assert sdk.endswith('.sdk')
     sdk =  sdk[6:-4]
-    return sdk
+    if sdk == '':
+        return get_os_level()
+    else:
+        return sdk
 
 
 
